src/QuartermasterShaman.sol

```diff
@@ -0,0 +1,226 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+// import { console2 } from "forge-std/Test.sol"; // remove before deploy
+import { HatsModule } from "hats-module/HatsModule.sol";
+import { IBaal } from "baal/interfaces/IBaal.sol";
+import { IBaalToken } from "baal/interfaces/IBaalToken.sol";
+
+/**
+ * @title Quartermaster Shaman
+ * @notice A Baal manager shaman that allows onboarding, offboarding, and other DAO member management
+ * by the holder of the captain hat. The captain uses the quartermaster to give crew status to new members,
+ * but there is a delay to avoid the captain gathering crew to avoid a mutiny.
```

Comment:
What is a mutiny?

Reply:
mutiny is just a proposal that transfers the captain hat to someone else. the crew needs the delay to allow for time to mutiny if the captain is up to any shenanigans.

Reply:
ah, nice!